manifests/m/Microsoft/WingetCreate/1.1.2.0/Microsoft.WingetCreate.locale.en-US.yaml

```
@@ -1,13 +1,13 @@
-# Created using wingetcreate 1.0.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
 
 PackageIdentifier: Microsoft.WingetCreate
 PackageVersion: 1.1.2.0
 PackageLocale: en-US
 Publisher: Microsoft Corporation
-PublisherUrl: https://www.microsoft.com/en-US/
+PublisherUrl: https://www.microsoft.com
 PublisherSupportUrl: https://github.com/microsoft/winget-create/issues
-PrivacyUrl: https://privacy.microsoft.com/en-US/privacystatement
+PrivacyUrl: https://github.com/microsoft/winget-create/blob/main/PRIVACY.md

```

```suggestion
PrivacyUrl: https://privacy.microsoft.com/en-US/privacystatement
```
The original URL is correct, as the GitHub one just tells users to go to the microsoft.com one